tests/internal/e2e/test/wdio.conf.ts

```diff
@@ -554,6 +511,84 @@ ENVIRO.END
         `cp ${path.join(repoRoot, "package.json")} ${extensionUnderTest}`
       );
     }
+
+    /**
+     * Prepares the execution of clicast and the creation of the CFG config file.
+     * @param initialWorkdir Path of the initial work dir.
+     */
+    async function prepareCVG(initialWorkdir: string): Promise<void> {
```

Comment:
Can we find a better name for this function, CVG is a bit cryptic

Reply:
Renamed prepareCVG to prepareConfig in 84eb017a6bc4c6acc21c3d9c578efc648f196ae6 .

Reply:
I'm curious: did `xo` not complain about that name anyway?

Reply:
We're not running xo on these for now, it also seems that the `xo` in the pipeline doesn't check E2E anyway. It's something to do later, in the "deal with all xo messages/configure xo" task

Reply:
Just run `xo --fix` on them, it will fix what it can; then spin through "obvious" messages on new suites (anything that's more than like 1 minute per issue, just ignore).

Reply:
... and now I realise this isn't a suite 😂